sky/packages/flx/lib/signing.dart

```diff
@@ -3,7 +3,7 @@
 // found in the LICENSE file.
 
 import 'dart:async';
-import 'dart:convert';
+import 'dart:convert' hide BASE64;
```

Comment:
I like special-casing this one, because I expect we'll revert this once BASE64 support makes it to the stable version of dart.